src/venn-client/index.ts

```diff
@@ -1,58 +1,64 @@
-import axios, { AxiosInstance } from 'axios';
-import { ethers, isAddress } from 'ethers';
+import axios, { AxiosInstance } from 'axios'
+import { TransactionRequest } from 'ethers'
 
-import { parseServerError } from '@/helpers';
-import { type SignedTxResponse, type SignTxServerRequest } from '@/types';
+import { errors } from '@/errors'
+import { isValidEthereumAddress, isValidUrl, parseApiError, parseServerError } from '@/helpers'
+import { type SignedTxResponse, type SignTxServerRequest } from '@/types'
 
 export type VennClientCreateOpts = {
-    vennURL: string;
-    vennPolicyAddress: string;
-    strict?: boolean;
-};
+    vennURL: string
+    vennPolicyAddress: string
+    isStrict?: boolean
```

Comment:
Let's leave this as `strict` please.
If you prefer, we can internally map this to a `isStrict` field, leaving the externally visible API as `strict`.

**Why?  (short version)**
Developers using this SDK and seeing the word `strict` will know right away what this does. Adding `isStrict` doesn't add tangible value, but makes the word longer to read.
Using `strict` is #good enough, and is more human-friendly.


**Why? (long version)**
My philosophy is that the intended audience for the code we write is human beings.
Consider the following snippet:
```javascript
const isLoggedIn = ...

if (isLoggedIn) {... }
```

Compared to this:
```javascript
const userIsLoggedIn = ...

if (userIsLoggedIn) {...}
```

The second version is much more readable to humans, and if you imagine your entire codebase written in this style _(while using ubiquitous language terms per your business domain)_ - the code becomes a delightful book to read by humans _(and the computer can still optimize it as needed)_.

Sure, I might get arrested by the "standards police" - but if we're being honest - did you ever encounter 2 code bases that use the same standards?   other than minor `isFoo`, every organization applies their own standard _(even backend vs frontend teams in the same org .. ahhem ahhem 😁)_


**Conclusion**
While I don't have a _(too)_ strong opinion on how our internal source code looks like - I have a strong opinion when it comes to code our end users / developers will use.

So, let's keep it `strict` please :)

Reply:
Can't say that I 100% agree 😅, but I'll leave it as 'strict'.